README.md

```diff
@@ -1,5 +1,13 @@
+<div align="center">
+  
 # ChatTTS
-[**English**](./README.md) | [**中文简体**](./README_CN.md)
+[**English**](./README.md) | [**中文简体**](./README_CN.md)     <div>
```

Comment:
- You should put the ipynb in this repo to ensure its security.
- The div-format is not so good.